packages/lesswrong/server/databaseSettings.ts

```diff
@@ -15,19 +14,11 @@ ensureIndex(DatabaseMetadata, {
   unique: true
 })
 
-if (!getServerSettingsLoaded())
-  void refreshSettingsCaches()
-
 if (isDevelopment && runValidateSettings) {
   // On development we validate the settings files, but wait 30 seconds to make sure that everything has really been loaded
   setTimeout(() => validateSettings(registeredSettings, getPublicSettings(), getServerSettingsCache()), 30000)
 }
 
-// We use Meteor.setInterval to make sure the code runs in a Fiber
-if (!isAnyTest) {
-  setInterval(refreshSettingsCaches, 1000 * 60 * 5) // We refresh the cache every 5 minutes on all servers
-}
-
```

Comment:
These existing here caused a very annoying dependency cycle and it doesn't actually seem like they absolutely had to live here.  Open to discussion.